homeassistant/components/fritzbox/sensor.py

```diff
@@ -65,6 +67,7 @@ class FritzSensorEntityDescription(
         native_unit_of_measurement=PERCENTAGE,
         device_class=DEVICE_CLASS_HUMIDITY,
         state_class=STATE_CLASS_MEASUREMENT,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
```

Comment:
Same.

Reply:
Is this the humidity of the plug? 😄

Reply:
Honestly, I'm not sure 🙈  here you might be right, this should be a primary entity 🤔  will change it soon